### PR TITLE
chore: show stopped tasks in `svc status`

### LINKED ISSUE #1947: Seperate stopped tasks from running tasks in `svc status` output
Hi everyone!

This issue is to propose a change to the current UI of `svc status`.
# Background

Below is the current UI for `svc status`.
```
Task Status

  ID                Image Digest        Last Status         Started At          Stopped At          Health Status
  --                ------------        -----------         ----------          ----------          -------------
  2032763a          8252e02b            RUNNING             1 hour ago          -                   UNKNOWN
  987548d7          8252e02b            RUNNING             1 hour ago          -                   HEALTHY
  e64eb63f          8252e02b            RUNNING             3 hours ago         -                   UNHEALTHY
```

Currently `$ svc status` shows tasks with "Running" as "Desired State", which doesn't capture any stopped tasks. This brings us the question of whether it makes sense to keep the "Stopped At" column for these tasks when the column is never filled.

With #1763 in perspective, "Stopped Reason" will also be included as a column. This will introduce one additional empty column to the table if `svc status` is run without `--with-stopped`.

# Proposal
Modify the current output of `svc status` to
```
Task Status

  ID                Image Digest        Last Status         Started At          Health Status
  --                ------------        -----------         ----------          -------------
  2032763c          8252e02b            RUNNING             1 hour ago          UNKNOWN
  987548d9          8252e02b            RUNNING             1 hour ago          HEALTHY
  e64eb63h          8252e02b            RUNNING             3 hours ago         UNHEALTHY
```

If `svc status --with-stopped` is run, the output will be
```
Task Status

  ID                Image Digest        Last Status         Started At          Health Status
  --                ------------        -----------         ----------          -------------
  2032763c          8252e02b            RUNNING             1 hour ago          UNKNOWN
  987548d9          8252e02b            RUNNING             1 hour ago          UNKNOWN
  e64eb63h          8252e02b            RUNNING             3 hours ago         UNKNOWN


Stopped Tasks

  ID                Image Digest        Last Status         Started At          Stopped At            Stopped Reason
  --                ------------        -----------         ----------          -------------         -------------
  3143763b          8252e02b            DEACTIVATING        2 hour ago          20 minutes ago        Essential container in task exited
  098548d8          8252e02b            DEACTIVATING        3 hour ago          20 minutes ago        Task stopped by user
  f75eb63g          8252e02b            STOPPED             4 hours ago         42 minutes ago        Task stopped by user
```

### PR DESCRIPTION
**Pull request by Lou1415926**
This PR shows all stopped tasks in a service when running `svc status`, as well as modifies the `svc status` output according to #1947.

This doesn't resolve #1947 since the [health check improvement](https://github.com/aws/copilot-cli/issues/1947#issuecomment-801290034) isn't addressed here.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/aws/ecs/task.go [hunk 1]
```diff
@@ -129,18 +129,40 @@ type TaskStatus struct {
...
+	if t.CapacityProvider != "" {
+		cp = t.CapacityProvider
+	}
+
+	return fmt.Sprintf("  %s\t%s\t%s\t%s\t%s\t%s\n", shortTaskID, imageDigest, t.LastStatus, startedSince, cp, taskHealthColor(t.Health))
+}
+
+// HumanString returns the stringified StoppedTaskStatus struct with human readable format.
+// Example output:
+//   6ca7a60d          f884127d            STOPPED             57 minutes ago             51 minutes ago
```
Comment (iamhopaul123) [inline internal/pkg/aws/ecs/task.go]: hmm should we include the stop reason in the example?

File: internal/pkg/aws/ecs/task_test.go [hunk 2]
```diff
@@ -296,6 +296,75 @@ func TestTaskStatus_HumanString(t *testing.T) {
...
+		"all params": {
+			health:        "HEALTHY",
+			id:            "aslhfnqo39j8qomimvoiqm89349",
+			lastStatus:    "RUNNING",
+			startedAt:     startTime,
+			stoppedAt:     stopTime,
+			imageDigest:   mockImageDigest,
+			stoppedReason: "Stopped by reasons",
+
+			wantTaskStatus: "  aslhfnqo\t18f7eb6c\tRUNNING\t14 years ago\t14 years ago\tStopped by reasons\n",
```
Comment (iamhopaul123) [inline internal/pkg/aws/ecs/task_test.go]: Can we show the result in ``` instead of " so that it would be more explicit?

**Review by iamhopaul123 — APPROVED** [review]
LGTM :shipit:

**Review by huanjani — APPROVED** [review]
Thanks for doing this!

**Pull request merged**